README.md

```diff
@@ -80,6 +80,17 @@ Launch analyze on flutter directory.
 
 Launch a Flutter command with custom arguments.
 
+* Set `arguments` to the argument string to pass to the flutter command.  Wrap the string in single quotes.
+* _(Optional)_. Select the `projectDirectory` that contains the `pubspec.yaml` file.
```

Comment:
add `(Optional). Set flutterDirectory to set path to the Flutter SDK if you were not using Flutter Install task before this one` here